src/api/verify.ts

```diff
@@ -114,31 +114,23 @@ class VerifyAPI {
     return trackRes;
   }
 
-  static async startTrackingVerified(params: RadarStartTrackingVerifiedParams) {
-    const doTrackVerified = async () => {
-      let trackRes;
-      try {
-        trackRes = await this.trackVerified(params);
-      } catch (err: any) {
-        Logger.error(`trackVerified error: ${err.message}`);
-      }
-      
+  static startTrackingVerified(params: RadarStartTrackingVerifiedParams) {
+    const scheduleNextIntervalWithLastToken = async () => {
       const { interval } = params;
 
-      let expiresIn = 0;
       let minInterval = interval;
 
-      if (trackRes) {
-        expiresIn = (trackRes.expiresIn || expiresIn);
+      if (lastToken) {
```

Comment:
Where is `lastToken` in scope here?

Reply:
Ah, I see it's defined at the top of the file.

Reply:
That's right. It stores the last `trackVerified()` response (if any)

Reply:
Is there a reason we refer to it as `lastToken` instead of lastResponse?

Reply:
Just mirroring var names in the mobile SDK to make it easy to compare across platforms. On the SDK, the response `{ passed, token, expiresIn, ... }` is deserialized to `RadarVerifiedLocationToken`